controllers/dataplane/openstackdataplanedeployment_controller.go

```diff
@@ -135,6 +133,8 @@ func (r *OpenStackDataPlaneDeploymentReconciler) Reconcile(ctx context.Context,
 				instance.Status.Conditions.Mirror(condition.ReadyCondition))
 		}
 
+		condition.RestoreLastTransitionTimes(
```

Comment:
Its probably not critical, but should this happen before the IsUnknown layered condition. If we restore the timestamps here, we'd not get the current timestamp of the mirror and instead the initial one. Or maybe thats what you want?

Reply:
its probably ok to have the initial timestamp when it went unknown